Gemfile.lock

```diff
@@ -72,8 +72,8 @@ GEM
     thread_safe (0.3.6)
     tzinfo (1.2.7)
       thread_safe (~> 0.1)
-    zeitwerk (2.3.0)
-    zxcvbn-ruby (1.0.0)
+    zeitwerk (2.4.0)
+    zxcvbn-ruby (1.1.0)
```

Comment:
### [zxcvbn-ruby](https://github.com/envato/zxcvbn-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.0 → 1.1.0

[_[change-log](https://github.com/envato/zxcvbn-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/envato/zxcvbn-ruby)_]





<details>
<summary>Commits</summary>

A change of **40** commits. See the full changes on [the compare page](https://github.com/envato/zxcvbn-ruby/compare/v1.0.0...v1.1.0).

These are the first 10 commits:
- (28c0745) [Don't shadow outer variable 'substitution'](https://github.com/envato/zxcvbn-ruby/commit/28c07456f87e2aae8f6932aab07a36e1dabcbd63)
- (5dd3879) [Guard would make testing easier](https://github.com/envato/zxcvbn-ruby/commit/5dd3879f3cd72426ab2d68d74e5de6f1ff293bf0)
- (b6f9924) [guard init](https://github.com/envato/zxcvbn-ruby/commit/b6f992454862cfacf56e8f2219265cb1c4efe903)
- (84d15b2) [Guard Bundler](https://github.com/envato/zxcvbn-ruby/commit/84d15b251500c90329659a0d42f8e6463e106ab6)
- (ddece55) [Guard RSpec](https://github.com/envato/zxcvbn-ruby/commit/ddece55d11f96d57cb2d603f6acee7f2c2f0df72)
- (b796875) [Tidy Guardfile](https://github.com/envato/zxcvbn-ruby/commit/b796875e3d9980742553c98606120776fb70b41d)
- (c4ff4bc) [Refactor dictionary spec to gain an understanding](https://github.com/envato/zxcvbn-ruby/commit/c4ff4bc63d0afaadbe9b195220759e947482f6fc)
- (abe3dec) [Feeling my way around the l33t_spec](https://github.com/envato/zxcvbn-ruby/commit/abe3dec4184e2d48e33593de74ec41cd3cd8cf95)
- (8dd27e5) [Tell us how it *is* expected to behave](https://github.com/envato/zxcvbn-ruby/commit/8dd27e599ae7bc8f900b536bf891cc885dcab0ec)
- (b60966f) [Handle invalid user dictionaries more robustly](https://github.com/envato/zxcvbn-ruby/commit/b60966f3fb6b16f7be04ccf04eedf49033013bbf)


</details>